services/director-v2/.env-devel

```diff
@@ -31,7 +31,7 @@ DIRECTOR_V2_GENERIC_RESOURCE_PLACEMENT_CONSTRAINTS_SUBSTITUTIONS='{}'
 
 LOG_LEVEL=DEBUG
 
-POSTGRES_ENDPOINT=${POSTGRES_ENDPOINT}
+POSTGRES_ENDPOINT=postgres:5432
```

Comment:
are you sure about this change in director-v2?

Reply:
I found this out when I tried to run the Director-v2 unit tests locally. It complains that the variable is referencing itself, as it is not defined earlier. I'm surprised nobody else has encountered this problem before.